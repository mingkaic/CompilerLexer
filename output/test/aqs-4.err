decafast(25685,0x7fff76133000) malloc: *** error for object 0x7ff363403900: pointer being freed was not allocated
*** set a breakpoint in malloc_error_break to debug
