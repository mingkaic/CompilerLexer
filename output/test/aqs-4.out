Program(None,Package(QuickSort,None,Method(displayList,VoidType,VarDef(start,IntType),VarDef(len,IntType),MethodBlock(VarDef(j,IntType),MethodCall(print_string,StringConstant("List:\n")),ForStmtAST(AssignVar(j,VariableExpr(start)),BinaryExpr(Lt,VariableExpr(j),BinaryExpr(Plus,VariableExpr(start),VariableExpr(len))),AssignVar(j,BinaryExpr(Plus,VariableExpr(j),NumberExpr(1)))),AssignVar(j,BinaryExpr(Plus,VariableExpr(j),NumberExpr(1)))))))
